src/author.rb

```diff
@@ -0,0 +1,20 @@
+class Author
+  attr_accessor :first_name, :last_name, :items
+  attr_reader :id
+
+  def initialize(first_name:, last_name:)
+    @id = Random.rand(1...1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    item.author = self
+    @items << item
+  end
+
+  def self.all
```

Comment:
Hi @Yonashenok  , 

![GIF](https://www.funimada.com/assets/images/cards/big/congrats-7.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

[OPTIONAL] it's worth mentioning that the use of ObjectSpace.each_object(self).to_a in the all method might not be the most efficient or recommended way to store and retrieve instances of the Author class. It would be more common to use a data structure like an array or a database to store and query instances of the Author class.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._